junit-jupiter-params/src/test/java/org/junit/jupiter/params/ParameterizedTestNameFormatterTests.java

```diff
@@ -60,6 +60,14 @@ void formatsDisplayName() {
 		assertEquals("enigma", formatter.format(2));
 	}
 
+	@Test
+	void formatsDisplayNameWithApostrophe() {
+		var formatter = formatter(DISPLAY_NAME_PLACEHOLDER, "display'Zero");
+
+		assertEquals("display'Zero", formatter.format(1));
+		assertEquals("display'Zero", formatter.format(2));
+	}
+
```

Comment:
```suggestion
	@Test
	void formatsDisplayNameWithApostrophe() {
		String displayName = "display'Zero";
		var formatter = formatter(DISPLAY_NAME_PLACEHOLDER, displayName);

		assertEquals(displayName, formatter.format(1));
		assertEquals(displayName, formatter.format(2));
	}

```

Please extract the `displayName` variable as above.

And please introduce the following test which demonstrates that the bug is not solely related to apostrophes.

```java
	@Test
	void formatsDisplayNameContainingFormatElements() {
		String displayName = "{enigma} {0} '{1}'";
		var formatter = formatter(DISPLAY_NAME_PLACEHOLDER, displayName);

		assertEquals(displayName, formatter.format(1));
		assertEquals(displayName, formatter.format(2));
	}
```